config/config-builder/src/key_manager_config.rs

```diff
@@ -24,10 +24,10 @@ impl Default for KeyManagerConfig {
     fn default() -> Self {
         let template = KMConfig::default();
         Self {
-            chain_id: ChainId::test(),
             rotation_period_secs: None,
             sleep_period_secs: None,
             txn_expiration_secs: None,
+            chain_id: ChainId::test(),
```

Comment:
the ordering here is a bit interesting... why not everything in alpha?

Reply:
This ordering matches the order of the field definitions (see above).